gojira.sh

```diff
@@ -277,6 +296,9 @@ function parse_args {
   if [[ $GOJIRA_MODE != "image" ]]; then
     GOJIRA_KONG_PATH=${GOJIRA_KONG_PATH:-$GOJIRA_KONGS/$PREFIX}
   fi
+
+  # Add CLI environment flags to Kong instance
+  [[ -n $_GOJIRA_CLI_ENVS ]] && add_egg cli_envs
```

Comment:
Last gotcha, I promise:

These conditional one liners are a double edged sword. When run at the end of a function, when the test fails the whole return code for the function also fails.

```
function lol {
  [[ $FOO ]] && echo HI
}


lol && echo "HELLO WORLD"
```

Reply:
Great catch!!!